sw/airborne/firmwares/rotorcraft/autopilot_arming_throttle.h

```diff
@@ -64,6 +65,30 @@ static inline void autopilot_arming_set(bool motors_on)
   }
 }
 
+/*Checks all arm requirements and returns false if OK and true otherwise.
+ * Also sets the arming status to provide information to the user
+ */
+static inline bool autopilot_arming_check_arm_requirements(void)
```

Comment:
This name isn't really quite logical with the description of the return. Additionally I would suggest reversing the logic, seems more intuitive. Eg. something like autopilot_arming_requirements_met(void)

Reply:
There is only one correct result (return 0 or false) but there can be multiple errors. (return 1=Error1, 2=Error_something_else, etc). 

So, this is logical, and as far as I know the standard.

Reply:
When a function returns a value, I think it is fine to have 0 for valid and non-zero for error codes. But since it returns a boolean, I would use true for valid and false if an error occurred.
What about `autopilot_arming_check_valid(void)` with the logic above?

Reply:
ok, the logic makes sense then but the return conditions then doesn't really follow from the function name (and return, uint8_t instead of bool). I dono just seems weird :p